app/views/admin/shared/_tabbed_navbar.erb

```diff
@@ -0,0 +1,20 @@
+<% nested_menu_items = AdminMenu.get_nested_menu_items(request.path.split("/")[-2], request.path.split("/")[-1]) %>
```

Comment:
This feels a bit wrong in the view, as indicated by the fact that we are performing the `split` twice. How about adding a new method to `AdminMenu`?

```ruby
def self.get_nested_menu_items_from_request(request)
  group, child_nav_item = request.path.split("/").last(2)
  get_nested_menu_item(group, child_nav_item)
end
```

This is one of the advantages of making the menu a model, we can add more useful methods to it as needed.

BTW, the `get_` prefix is not particularly idiomatic in Ruby, I suggest `nested_menu_items` and `nested_menu_items_from_request`.

Reply:
This makes sense, I've addressed your feedback ✅